Index: app/src/main/res/drawable/button_use_dark_gray.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/drawable/button_use_dark_gray.xml	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/res/drawable/button_use_dark_gray.xml	(revision )
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
     <corners
-        android:radius="3dp"/>
+        android:radius="0dp"/>
     <solid
         android:color="@color/myPrimaryColor"/>
 </shape>
\ No newline at end of file
Index: app/src/main/java/pupccb/solutionsresource/com/adapter/TicketAdapter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pupccb/solutionsresource/com/adapter/TicketAdapter.java	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/java/pupccb/solutionsresource/com/adapter/TicketAdapter.java	(revision )
@@ -1,5 +1,6 @@
 package pupccb.solutionsresource.com.adapter;
 
+import android.graphics.Color;
 import android.support.v7.widget.CardView;
 import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
@@ -34,9 +35,17 @@
 
     @Override
     public void onBindViewHolder(TicketViewHolder ticketViewHolder, int i) {
+        if(tickets.get(i).status.equalsIgnoreCase("Open")){
+            ticketViewHolder.status.setBackgroundResource(R.color.open);
+        }else if(tickets.get(i).status.equalsIgnoreCase("Resolved")){
+            ticketViewHolder.status.setBackgroundResource(R.color.resolved);
+        }else if(tickets.get(i).status.equalsIgnoreCase("Ongoing")){
+            ticketViewHolder.status.setBackgroundResource(R.color.ongoing);
+        }
         ticketViewHolder.status.setText(tickets.get(i).status);
         ticketViewHolder.title.setText(tickets.get(i).title);
         ticketViewHolder.assignTo.setText(tickets.get(i).assignTo);
+
     }
 
     @Override
@@ -51,16 +60,17 @@
 
     public static class TicketViewHolder extends RecyclerView.ViewHolder {
         CardView cv;
-        TextView status;
         TextView title;
         TextView assignTo;
+        TextView status;
 
+
         TicketViewHolder(View itemView) {
             super(itemView);
             cv = (CardView)itemView.findViewById(R.id.ticket_info_card);
-            status = (TextView)itemView.findViewById(R.id.ticketDetailStatusTextView);
             title = (TextView)itemView.findViewById(R.id.ticketSubjectTextView);
             assignTo = (TextView)itemView.findViewById(R.id.assignedToTextView);
+            status = (TextView)itemView.findViewById(R.id.ticketDetailStatusTextView);
         }
     }
 
Index: app/src/main/java/pupccb/solutionsresource/com/model/RegistrationResponse.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pupccb/solutionsresource/com/model/RegistrationResponse.java	(revision )
+++ app/src/main/java/pupccb/solutionsresource/com/model/RegistrationResponse.java	(revision )
@@ -0,0 +1,20 @@
+package pupccb.solutionsresource.com.model;
+
+/**
+ * Created by User on 8/14/2015.
+ */
+public class RegistrationResponse {
+
+    private String message, description;
+
+    public String getMessage() {
+        return message != null ? message.trim() : "";
+    }
+    public String getDescription() {
+        return description != null ? description.trim() : "";
+    }
+    public void setDescription(String description) {
+        this.description = description != null ? description.trim() : "";;
+    }
+
+}
Index: app/src/main/java/pupccb/solutionsresource/com/model/Ticket.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pupccb/solutionsresource/com/model/Ticket.java	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/java/pupccb/solutionsresource/com/model/Ticket.java	(revision )
@@ -7,10 +7,12 @@
     public String status;
     public String title;
     public String assignTo;
+    public int color;
 
-    public Ticket(String status, String title, String assignTo) {
+    public Ticket(String status, String title, String assignTo, int color) {
         this.status = status;
         this.title = title;
         this.assignTo =assignTo;
+        this.color = color;
     }
 }
Index: app/src/main/java/pupccb/solutionsresource/com/helper/request/SessionRequest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pupccb/solutionsresource/com/helper/request/SessionRequest.java	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/java/pupccb/solutionsresource/com/helper/request/SessionRequest.java	(revision )
@@ -21,14 +21,10 @@
     @Override
     public Session loadDataFromNetwork() throws Exception {
         return getService().getAccessToken(login.getUsername(),
-                login.getPassword(),
-                login.getClient_id(),
-                login.getClient_secret(),
-                login.getGrant_type(),
-                login.getScope());
+                login.getPassword());
     }
 
     public String createCacheKey() {
-        return "SessionRequest" + login.getClient_secret();
+        return "SessionRequest" + login.getUsername();
     }
 }
Index: app/src/main/java/pupccb/solutionsresource/com/helper/service/RegistrationService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pupccb/solutionsresource/com/helper/service/RegistrationService.java	(revision )
+++ app/src/main/java/pupccb/solutionsresource/com/helper/service/RegistrationService.java	(revision )
@@ -0,0 +1,25 @@
+package pupccb.solutionsresource.com.helper.service;
+
+
+import pupccb.solutionsresource.com.model.RegistrationResponse;
+import pupccb.solutionsresource.com.model.Session;
+import retrofit.http.Field;
+import retrofit.http.FormUrlEncoded;
+import retrofit.http.POST;
+
+/**
+ * Created by User on 7/16/2015.
+ */
+public interface RegistrationService {
+
+    @FormUrlEncoded
+    @POST("/")
+    RegistrationResponse register(
+            @Field("firsname") String firstname,
+            @Field("middlename") String middlename,
+            @Field("lastname") String lastname,
+            @Field("contactnumber") String contactnumber,
+            @Field("emailaddress") String emailaddress,
+            @Field("password") String passsword
+    );
+}
Index: app/src/main/res/drawable/button_use_light_gray.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/drawable/button_use_light_gray.xml	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/res/drawable/button_use_light_gray.xml	(revision )
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
     <corners
-        android:radius="5dp"/>
+        android:radius="0dp"/>
     <solid
         android:color="@color/txt"/>
 </shape>
\ No newline at end of file
Index: app/src/main/java/pupccb/solutionsresource/com/fragment/MyTicket.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pupccb/solutionsresource/com/fragment/MyTicket.java	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/java/pupccb/solutionsresource/com/fragment/MyTicket.java	(revision )
@@ -51,7 +51,7 @@
         viewPager.setAdapter(myTicketTabPagerAdapter);
         PagerSlidingTabStrip pagerSlidingTabStrip = (PagerSlidingTabStrip) view.findViewById(R.id.pagerSlidingTabStrip);
         pagerSlidingTabStrip.setViewPager(viewPager);
-        pagerSlidingTabStrip.setIndicatorColor(getResources().getColor(R.color.main_color_gray_lt));
+        pagerSlidingTabStrip.setIndicatorColor(getResources().getColor(R.color.white));
         pagerSlidingTabStrip.setTextColor(getResources().getColor(R.color.white));
 
 //        viewPager.setOnPageChangeListener(new ViewPager.OnPageChangeListener() {
Index: app/src/main/res/layout/activity_registration.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/activity_registration.xml	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/res/layout/activity_registration.xml	(revision )
@@ -69,7 +69,7 @@
     </EditText>
 
     <EditText
-        android:id="@+id/editTextConactNumber"
+        android:id="@+id/editTextContactNumber"
         style="@style/editext_graybg"
         android:layout_width="fill_parent"
         android:layout_height="@dimen/pad_40dp"
@@ -83,7 +83,6 @@
         style="@style/editext_graybg"
         android:layout_width="fill_parent"
         android:layout_height="@dimen/pad_60dp"
-        android:drawableLeft="@mipmap/ic_username"
         android:layout_marginTop="@dimen/pad_5dp"
         android:hint="@string/email">
 
@@ -99,7 +98,6 @@
         android:layout_height="@dimen/pad_60dp"
         android:layout_marginBottom="@dimen/pad_5dp"
         android:layout_marginTop="@dimen/pad_5dp"
-        android:drawableLeft="@mipmap/ic_pwd"
         android:hint="@string/password"
         android:inputType="textPassword" />
     <EditText
@@ -108,13 +106,12 @@
         android:layout_width="fill_parent"
         android:layout_height="@dimen/pad_60dp"
         android:layout_marginBottom="@dimen/pad_10dp"
-        android:drawableLeft="@mipmap/ic_pwd"
         android:hint="@string/confirm_password"
         android:inputType="textPassword" />
 
 
     <Button
-        android:id="@+id/btnLogin"
+        android:id="@+id/btnSignUp"
         style="@style/btn_white_large"
         android:layout_width="fill_parent"
         android:layout_height="@dimen/pad_60dp"
\ No newline at end of file
Index: app/src/main/res/values/colors.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/values/colors.xml	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/res/values/colors.xml	(revision )
@@ -8,13 +8,13 @@
         http://www.google.com/design/spec/style/color.html#color-ui-color-palette
     -->
 
-    <color name="myPrimaryColor">#49a3d7</color>
-    <color name="myPrimaryDarkColor">#e5664f</color>
-    <color name="myAccentColor">#E9BC1B</color>
+    <color name="myPrimaryColor">#0d509b</color>
+    <color name="myPrimaryDarkColor">#114274</color>
+    <color name="myAccentColor">#ba0015</color>
 
     <color name="myTextPrimaryColor">#313335</color>
 
-    <color name="myDrawerBackground">#fdfdfe</color>
+    <color name="myDrawerBackground">#ffffffff</color>
     <color name="myWindowBackground">#fdfdfe</color>
 
     <color name="myNavigationColor">#000</color>
@@ -54,7 +54,7 @@
     <color name="text_white_87">#deFFFFFF</color>
     <color name="background_card">#fff</color>
     <color name="txt">#e9e3f1</color>
-    <color name="background">#fdfdcb</color>
+    <color name="ticket_info_txt">#d5dee5</color>
 
     <array name="note_neutral_colors">
         <item>#9E9E9E</item>
@@ -79,11 +79,15 @@
     <color name="social_button_label">#6D6D6D</color>
     <color name="social_button_count">#bdbdbd</color>
 
-    <color name="tab_color">@color/myPrimaryColor</color>
+    <color name="tab_color">@color/myPrimaryDarkColor</color>
     <color name="view_pager_color">#FFE5E5E5</color>
 
 
 
+    <!-- Ticket Color Status-->
+    <color name="open">#73d2f0</color>
+    <color name="resolved">#f2d560</color>
+    <color name="ongoing">#83d083</color>
     <!--theme-->
     <color name="red">#B81D18</color>
     <color name="blue">#004687</color>
\ No newline at end of file
Index: app/src/main/res/layout/activity_login.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/activity_login.xml	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/res/layout/activity_login.xml	(revision )
@@ -2,40 +2,39 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="fill_parent"
     android:layout_height="fill_parent"
+    android:background="@drawable/portrait_bg"
     android:gravity="center_horizontal"
     android:orientation="vertical"
     android:paddingLeft="@dimen/pad_20dp"
-    android:paddingRight="@dimen/pad_20dp"
+    android:paddingRight="@dimen/pad_20dp">
 
-    >
 
-
     <ImageView
-        android:id="@id/imageView1"
+        android:id="@+id/imageView1"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_margin="@dimen/pad_5dp"
         android:layout_weight="1.0"
         android:src="@drawable/ccb" />
 
 
     <!--<EditText-->
-        <!--android:id="@+id/editTextSecret"-->
-        <!--style="@style/editext_graybg"-->
-        <!--android:layout_width="fill_parent"-->
-        <!--android:layout_height="@dimen/pad_60dp"-->
-        <!--android:layout_marginBottom="@dimen/pad_5dp"-->
-        <!--android:layout_marginTop="@dimen/pad_5dp"-->
-        <!--android:drawableLeft="@mipmap/ic_pwd"-->
-        <!--android:hint="@string/client_secret" />-->
+    <!--android:id="@+id/editTextSecret"-->
+    <!--style="@style/editext_graybg"-->
+    <!--android:layout_width="fill_parent"-->
+    <!--android:layout_height="@dimen/pad_60dp"-->
+    <!--android:layout_marginBottom="@dimen/pad_5dp"-->
+    <!--android:layout_marginTop="@dimen/pad_5dp"-->
+    <!--android:drawableLeft="@mipmap/ic_pwd"-->
+    <!--android:hint="@string/client_secret" />-->
 
     <EditText
-        android:id="@+id/editTextUsername"
+        android:id="@+id/editTextEmail"
         style="@style/editext_graybg"
         android:layout_width="fill_parent"
         android:layout_height="@dimen/pad_60dp"
         android:drawableLeft="@mipmap/ic_username"
-        android:hint="@string/email"
-        >
+        android:hint="@string/email">
 
         <requestFocus />
     </EditText>
@@ -57,29 +56,31 @@
         style="@style/btn_white_large"
         android:layout_width="fill_parent"
         android:layout_height="@dimen/pad_60dp"
+        android:layout_marginBottom="@dimen/pad_15dp"
         android:text="@string/log_in"
         android:textColor="@color/white" />
 
     <TextView
-        android:id="@id/btnReg"
-        android:clickable="true"
+        android:id="@+id/btnReg"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/pad_10dp"
+        android:layout_marginTop="@dimen/pad_5dp"
+        android:clickable="true"
 
         android:text="@string/register"
-        android:textColor="@color/myPrimaryColor"
-        android:textSize="@dimen/txt_14sp"
+        android:textColor="@color/white"
+        android:textSize="@dimen/txt_15sp"
         android:textStyle="bold" />
 
     <TextView
         android:id="@id/btnForgot"
-        android:clickable="true"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:layout_marginBottom="@dimen/pad_20dp"
         android:layout_marginTop="@dimen/pad_5dp"
-        android:layout_marginBottom="@dimen/pad_10dp"
+        android:clickable="true"
         android:text="@string/forgot_my_pwd"
-        android:textColor="@color/myPrimaryColor"
+        android:textColor="@color/white"
         android:textSize="@dimen/txt_12sp" />
 </LinearLayout>
Index: app/src/main/java/pupccb/solutionsresource/com/activity/Registration.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pupccb/solutionsresource/com/activity/Registration.java	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/java/pupccb/solutionsresource/com/activity/Registration.java	(revision )
@@ -2,27 +2,60 @@
 
 
 import android.content.Intent;
+import android.content.SharedPreferences;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
 import android.view.View;
-import android.widget.Spinner;
-import android.widget.TextView;
+import android.widget.EditText;
+import android.widget.Toast;
 
-import java.util.ArrayList;
+import com.mobsandgeeks.saripaar.Rule;
+import com.mobsandgeeks.saripaar.ValidationError;
+import com.mobsandgeeks.saripaar.Validator;
+import com.mobsandgeeks.saripaar.annotation.ConfirmPassword;
+import com.mobsandgeeks.saripaar.annotation.CreditCard;
+import com.mobsandgeeks.saripaar.annotation.Digits;
+import com.mobsandgeeks.saripaar.annotation.Email;
+import com.mobsandgeeks.saripaar.annotation.NotEmpty;
+import com.mobsandgeeks.saripaar.annotation.Password;
+import com.mobsandgeeks.saripaar.annotation.Pattern;
 
+import java.util.List;
+
 import pupccb.solutionsresource.com.R;
+import pupccb.solutionsresource.com.helper.Controller;
+import pupccb.solutionsresource.com.helper.OnlineHelper;
+import pupccb.solutionsresource.com.model.RegistrationDetails;
+import pupccb.solutionsresource.com.model.RegistrationResponse;
+import pupccb.solutionsresource.com.util.TouchEffect;
 
 /**
  * Created by User on 8/6/2015.
  */
-public class Registration extends AppCompatActivity {
+public class Registration extends AppCompatActivity implements View.OnClickListener, Validator.ValidationListener {
 
+
+    public static final TouchEffect TOUCH = new TouchEffect();
+    private Controller controller;
+    private Validator validator;
+    private SharedPreferences sharedPreferences;
     private Toolbar toolbar;
-    private ArrayList<String> arraylist;
-    private Spinner spinner;
     private View view;
-    private TextView txtHaveAccount;
+    @NotEmpty
+    private EditText editTextFname;
+    @NotEmpty
+    private EditText editTextMname;
+    @NotEmpty
+    private EditText editTextLname;
+    @NotEmpty
+    private EditText editTextContactNumber;
+    @Email
+    private EditText editTextEmail;
+    @Password(min = 6, scheme = Password.Scheme.ALPHA_NUMERIC, message = "Your password must contain at least 6 alphanumeric characters")
+    private EditText editTextPassword;
+    @ConfirmPassword
+    private EditText editTextConfirmPassword;
 
 
     @Override
@@ -30,32 +63,92 @@
         super.onCreate(savedInstanceState);
         view = getLayoutInflater().inflate(R.layout.activity_registration, null);
         setContentView(view);
-        toolbar = (Toolbar) findViewById(R.id.toolbar);
+        toolBar(view);
+        findViewById(view);
+        startController();
+    }
 
+    private void toolBar(View view) {
+        toolbar = (Toolbar) view.findViewById(R.id.toolbar);
         if (toolbar != null) {
             setSupportActionBar(toolbar);
             getSupportActionBar().setDisplayHomeAsUpEnabled(true);
         }
-        findViewById(view);
+    }
 
+    private void startController() {
+        controller = new Controller(new OnlineHelper());
     }
 
     private void findViewById(View view) {
+        validator = new Validator(this);
+        validator.setValidationListener(this);
+        setTouchNClick(R.id.btnHaveAccount);
+        setTouchNClick(R.id.btnSignUp);
 
-        txtHaveAccount = (TextView) view.findViewById(R.id.btnHaveAccount);
-        txtHaveAccount.setOnClickListener(buttonListener);
+        editTextFname = (EditText) view.findViewById(R.id.editTextFname);
+        editTextMname = (EditText) view.findViewById(R.id.editTextMname);
+        editTextLname = (EditText) view.findViewById(R.id.editTextLname);
+        editTextContactNumber = (EditText) view.findViewById(R.id.editTextContactNumber);
+        editTextEmail = (EditText) view.findViewById(R.id.editTextEmail);
+        editTextPassword = (EditText) view.findViewById(R.id.editTextPassword);
+        editTextConfirmPassword = (EditText) view.findViewById(R.id.editTextConfirmPassword);
 
     }
 
-    private View.OnClickListener buttonListener = new View.OnClickListener() {
-        @Override
+    public void register(RegistrationDetails registrationDetails) {
+        controller.register(this, registrationDetails);
+    }
+
+    public void registerResult(RegistrationResponse registrationResponse, RegistrationDetails registrationDetails) {
+        finish();
+        startActivity(new Intent(Registration.this, Main.class));
+    }
+
-        public void onClick(View view) {
+    public void onClick(View view) {
-            if(view == txtHaveAccount){
+        if (view.getId() == R.id.btnHaveAccount) {
-                startActivity(new Intent(getApplicationContext(), Main.class));
-                finish();
+            startActivity(new Intent(getApplicationContext(), Main.class));
+            finish();
+        } else if (view.getId() == R.id.btnSignUp) {
+            validator.validate();
+//            startActivity(new Intent(getApplicationContext(), Main.class));
+//            finish();
-            }
-        }
+        }
+    }
-    };
+
+    @Override
+    public void onValidationSucceeded() {
+        register(new RegistrationDetails("", "", "", "", "", ""));
+    }
+
+
+    @Override
+    public void onValidationFailed(List<ValidationError> errors) {
+        for (ValidationError error : errors) {
+            View view = error.getView();
+            String message = error.getCollatedErrorMessage(this);
+
+            // Display error messages ;)
+            if (view instanceof EditText) {
+                ((EditText) view).setError(message);
+            } else {
+                Toast.makeText(this, message, Toast.LENGTH_LONG).show();
+            }
+        }
+    }
+
+
+    public View setClick(int btn) {
+        View view = this.findViewById(btn);
+        view.setOnClickListener(this);
+        return view;
+    }
+
+    public View setTouchNClick(int btn) {
+        View view = this.setClick(btn);
+        view.setOnTouchListener(TOUCH);
+        return view;
+    }
 
     @Override
     public void onBackPressed() {
Index: app/src/main/java/pupccb/solutionsresource/com/helper/communicator/OnlineCommunicator.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pupccb/solutionsresource/com/helper/communicator/OnlineCommunicator.java	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/java/pupccb/solutionsresource/com/helper/communicator/OnlineCommunicator.java	(revision )
@@ -4,6 +4,7 @@
 
 import pupccb.solutionsresource.com.helper.Controller;
 import pupccb.solutionsresource.com.model.Login;
+import pupccb.solutionsresource.com.model.RegistrationDetails;
 
 /**
  * Created by User on 7/16/2015.
@@ -11,4 +12,5 @@
 public interface OnlineCommunicator {
 
     void login(Controller controller, Activity activity, Login login, Controller.MethodTypes methodTypes);
+    void register(Controller controller, Activity activity, RegistrationDetails registrationDetails, Controller.MethodTypes methodTypes);
 }
Index: app/app.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/app.iml	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/app.iml	(revision )
@@ -96,19 +96,19 @@
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" exported="" name="design-22.2.1" level="project" />
     <orderEntry type="library" exported="" name="robospice-cache-1.4.12" level="project" />
-    <orderEntry type="library" exported="" name="android-saripaar-1.0.3" level="project" />
-    <orderEntry type="library" exported="" name="robospice-retrofit-1.4.12" level="project" />
     <orderEntry type="library" exported="" name="picasso-2.3.4" level="project" />
+    <orderEntry type="library" exported="" name="robospice-retrofit-1.4.12" level="project" />
     <orderEntry type="library" exported="" name="pagerslidingtabstrip-1.0.1" level="project" />
     <orderEntry type="library" exported="" name="gson-2.2.4" level="project" />
     <orderEntry type="library" exported="" name="support-v4-22.2.1" level="project" />
+    <orderEntry type="library" exported="" name="android-saripaar-2.0.2" level="project" />
     <orderEntry type="library" exported="" name="recyclerview-v7-22.2.1" level="project" />
     <orderEntry type="library" exported="" name="cardview-v7-22.2.1" level="project" />
     <orderEntry type="library" exported="" name="support-annotations-22.2.1" level="project" />
     <orderEntry type="library" exported="" name="appcompat-v7-22.2.1" level="project" />
-    <orderEntry type="library" exported="" name="commons-io-1.3.2" level="project" />
     <orderEntry type="library" exported="" name="retrofit-1.5.0" level="project" />
-    <orderEntry type="library" exported="" name="commons-lang3-3.3.2" level="project" />
+    <orderEntry type="library" exported="" name="commons-io-1.3.2" level="project" />
     <orderEntry type="library" exported="" name="robospice-1.4.12" level="project" />
+    <orderEntry type="library" exported="" name="commons-lang3-3.3.2" level="project" />
   </component>
 </module>
\ No newline at end of file
Index: app/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/build.gradle	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/build.gradle	(revision )
@@ -33,7 +33,7 @@
     compile 'com.android.support:appcompat-v7:22.2.1'
     compile 'com.android.support:recyclerview-v7:22.+'
     compile 'com.android.support:cardview-v7:22.+'
-    compile 'com.mobsandgeeks:android-saripaar:1.0.3'
+    compile 'com.mobsandgeeks:android-saripaar:2.0.2'
     compile 'com.octo.android.robospice:robospice:1.4.12'
     compile 'com.octo.android.robospice:robospice-retrofit:1.4.12'
     compile 'com.squareup.picasso:picasso:2.3.4'
Index: app/src/main/java/pupccb/solutionsresource/com/model/RegistrationDetails.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pupccb/solutionsresource/com/model/RegistrationDetails.java	(revision )
+++ app/src/main/java/pupccb/solutionsresource/com/model/RegistrationDetails.java	(revision )
@@ -0,0 +1,66 @@
+package pupccb.solutionsresource.com.model;
+
+/**
+ * Created by User on 8/14/2015.
+ */
+public class RegistrationDetails {
+
+    private String firstname, middlename, lasrname, contactnumber, emailaddress, password;
+
+    public RegistrationDetails(String firstname, String middlename, String lasrname, String contactnumber, String emailaddress,String password) {
+        setFirstname(firstname);
+        setMiddlename(middlename);
+        setLasrname(lasrname);
+        setContactnumber(contactnumber);
+        setEmailaddress(emailaddress);
+        setPassword(password);
+    }
+
+    public String getFirstname() {
+        return firstname != null ? firstname.trim() : "";
+    }
+
+    public String getMiddlename() {
+        return middlename != null ? middlename.trim() : "";
+    }
+
+    public String getLasrname() {
+        return lasrname != null ? lasrname.trim() : "";
+    }
+
+    public String getContactnumber() {
+        return contactnumber != null ? contactnumber.trim() : "";
+    }
+
+    public String getEmailaddress() {
+        return emailaddress != null ? emailaddress.trim() : "";
+    }
+
+    public String getPassword() {
+        return password != null ? password.trim() : "";
+    }
+
+    public void setFirstname(String firstname) {
+        this.firstname = firstname != null ? firstname.trim() : "";
+    }
+
+    public void setMiddlename(String middlename) {
+        this.middlename = middlename != null ? middlename.trim() : "";
+    }
+
+    public void setLasrname(String lasrname) {
+        this.lasrname = lasrname != null ? lasrname.trim() : "";
+    }
+
+    public void setContactnumber(String contactnumber) {
+        this.contactnumber = contactnumber != null ? contactnumber.trim() : "";
+    }
+
+    public void setEmailaddress(String emailaddress) {
+        this.emailaddress = emailaddress != null ? emailaddress.trim() : "";
+    }
+
+    public void setPassword(String password) {
+        this.password = password != null ? password.trim() : "";
+    }
+}
Index: app/src/main/java/pupccb/solutionsresource/com/model/Login.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pupccb/solutionsresource/com/model/Login.java	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/java/pupccb/solutionsresource/com/model/Login.java	(revision )
@@ -5,64 +5,28 @@
  */
 public class Login {
 
-    private String username, password, client_id, client_secret, scope, grant_type;
+    private String username, password;
 
-    public Login(String username, String password, String client_id,
-                 String client_secret, String scope, String grant_type) {
+    public Login(String username, String password) {
         setUsername(username);
         setPassword(password);
-        setClient_id(client_id);
-        setClient_secret(client_secret);
-        setScope(scope);
-        setGrant_type(grant_type);
     }
 
     public String getUsername() {
         return username != null ? username.trim() : "";
     }
 
-    public String getPassword() {
-        return password != null ? password.trim() : "";
-    }
-
-    public String getClient_id() {
-        return client_id != null ? client_id.trim() : "";
-    }
-
-    public String getClient_secret() {
-        return client_secret != null ? client_secret.trim() : "";
-    }
-
-    public String getScope() {
-        return scope != null ? scope.trim() : "";
-    }
-
-    public String getGrant_type() {
-        return grant_type != null ? grant_type.trim() : "";
-    }
-
     public void setUsername(String username) {
         this.username = username != null ? username.trim() : "";
     }
 
+    public String getPassword() {
+        return password != null ? password.trim() : "";
+    }
+
     public void setPassword(String password) {
         this.password = password != null ? password.trim() : "";
     }
 
-    public void setClient_id(String client_id) {
-        this.client_id = client_id != null ? client_id.trim() : "";
-    }
-
-    public void setClient_secret(String client_secret) {
-        this.client_secret = client_secret != null ? client_secret.trim() : "";
-    }
-
-    public void setScope(String scope) {
-        this.scope = scope != null ? scope.trim() : "";
-    }
-
-    public void setGrant_type(String grant_type) {
-        this.grant_type = grant_type != null ? grant_type.trim() : "";
-    }
 
 }
Index: app/src/main/res/values/strings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/values/strings.xml	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/res/values/strings.xml	(revision )
@@ -10,7 +10,7 @@
     <string name="menu_item_one">Menu Item</string>
 
 
-    <string name="forgot_my_pwd">I\'ve forgotten my password?</string>
+    <string name="forgot_my_pwd">Forgot your password?</string>
     <string name="already_have_account">Already have an account? Sign In</string>
     <string name="client_secret">Client Secret</string>
     <string name="username">Username</string>
Index: app/src/main/res/layout/ticket_info_card_view.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/ticket_info_card_view.xml	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/res/layout/ticket_info_card_view.xml	(revision )
@@ -9,15 +9,7 @@
 
         xmlns:card_view="http://schemas.android.com/apk/res-auto"
         android:id="@+id/ticket_info_card"
-        android:layout_width="fill_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="5dip"
-        android:layout_marginLeft="4.0dip"
-        android:layout_marginRight="4.0dip"
-        android:layout_marginTop="5dip"
-        android:foreground="?selectableItemBackground"
-        card_view:cardCornerRadius="4dp"
-        card_view:cardElevation="4dp"
+        style="@style/Widget.MaterialSheetFab.Ticket"
         android:clickable="true"
         >
 
@@ -27,18 +19,18 @@
             android:layout_height="fill_parent"
             android:layout_alignParentLeft="true"
             android:layout_alignParentTop="true"
-            android:background="@color/view_pager_color">
+            android:background="@color/white">
 
             <TextView
                 android:id="@+id/ticketDetailStatusTextView"
                 style="@style/TEXT.MEDIUM"
                 android:layout_width="wrap_content"
                 android:layout_height="28.0dip"
-                android:layout_alignParentLeft="true"
+                android:layout_alignParentRight="true"
                 android:layout_alignParentTop="true"
                 android:layout_marginBottom="4.0dip"
                 android:layout_marginTop="8.0dip"
-                android:background="#ffff0000"
+                android:background="@color/myAccentColor"
                 android:gravity="left|center"
                 android:maxLines="1"
                 android:paddingLeft="10.0dip"
@@ -53,37 +45,33 @@
                 android:id="@+id/ticketDetailLastRespondedImageView"
                 android:layout_width="14.0dip"
                 android:layout_height="14.0dip"
-                android:layout_alignBottom="@+id/ticketDetailStatusTextView"
-                android:layout_alignParentRight="true"
+                android:src="@drawable/icon_gray_dot"
+                android:visibility="invisible"
                 android:layout_alignTop="@+id/ticketDetailStatusTextView"
-                android:layout_centerVertical="true"
-                android:layout_marginLeft="6.0dip"
-                android:layout_marginRight="6.0dip"
-                android:layout_marginTop="4.0dip"
-                android:src="@drawable/icon_gray_dot" />
+                android:layout_toLeftOf="@+id/ticketDetailStatusTextView"
+                android:layout_toStartOf="@+id/ticketDetailStatusTextView"
+                android:layout_alignBottom="@+id/ticketDetailStatusTextView" />
 
             <TextView
                 android:id="@+id/ticketSubjectTextView"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_alignParentLeft="true"
-                android:layout_below="@+id/ticketDetailStatusTextView"
-                android:layout_marginBottom="8.0dip"
-                android:layout_marginLeft="14.0dip"
-                android:layout_marginRight="14.0dip"
-                android:layout_marginTop="8.0dip"
                 android:ellipsize="end"
                 android:maxLines="3"
                 android:text="Title"
-                android:textColor="#ff000000"
-                android:textSize="24.0sp" />
+                android:textColor="@color/black"
+                android:textSize="@dimen/pad_15dp"
+                android:layout_alignTop="@+id/ticketDetailLastRespondedImageView"
+                android:layout_alignLeft="@+id/informationLinearLayout"
+                android:layout_alignStart="@+id/informationLinearLayout" />
 
             <LinearLayout
                 android:id="@+id/informationLinearLayout"
                 android:layout_width="fill_parent"
                 android:layout_height="wrap_content"
                 android:layout_below="@+id/ticketSubjectTextView"
-                android:layout_marginBottom="20.0dip"
+                android:layout_marginTop="15.0dip"
+                android:layout_marginBottom="15.0dip"
                 android:layout_marginLeft="16.0dip"
                 android:layout_marginRight="16.0dip"
                 android:orientation="horizontal">
@@ -157,7 +145,7 @@
                 android:layout_width="fill_parent"
                 android:layout_height="wrap_content"
                 android:layout_below="@id/cardDivider"
-                android:background="?selectableItemBackground"
+                android:background="@color/ticket_info_txt"
                 android:orientation="horizontal">
 
                 <TextView
@@ -169,7 +157,7 @@
                     android:layout_marginTop="16.0dip"
                     android:layout_weight="1.0"
                     android:text="@string/title_activity_ticket_info"
-                    android:textColor="@color/myAccentColor"
+                    android:textColor="#273540"
                     android:textSize="14.0sp" />
 
                 <ImageView
@@ -177,7 +165,8 @@
                     android:layout_height="24.0dip"
                     android:layout_gravity="center_vertical"
                     android:layout_marginRight="4.0dip"
-                    android:src="@drawable/chevron_right" />
+                    android:src="@drawable/chevron_right_black"
+                     />
             </LinearLayout>
         </RelativeLayout>
     </android.support.v7.widget.CardView>
\ No newline at end of file
Index: app/src/main/java/pupccb/solutionsresource/com/helper/service/SessionService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pupccb/solutionsresource/com/helper/service/SessionService.java	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/java/pupccb/solutionsresource/com/helper/service/SessionService.java	(revision )
@@ -15,9 +15,10 @@
     @POST("/oauth/access_token")
     Session getAccessToken(
             @Field("username") String username,
-            @Field("password") String password,
-            @Field("client_id") String client_id,
-            @Field("client_secret") String client_secret,
-            @Field("grant_type") String grant_type,
-            @Field("scope") String scope);
+            @Field("password") String password
+//            @Field("client_id") String client_id,
+//            @Field("client_secret") String client_secret,
+//            @Field("grant_type") String grant_type,
+//            @Field("scope") String scope
+            );
 }
Index: app/src/main/AndroidManifest.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/AndroidManifest.xml	(revision cf1215e63aa386c56ae6fa7450c1b0aec0b4b9c2)
+++ app/src/main/AndroidManifest.xml	(revision )
@@ -21,8 +21,8 @@
             android:exported="false" />
 
         <activity  android:name=".activity.Main"
-            android:screenOrientation="portrait"
             android:label="@string/app_name"
+
             android:windowSoftInputMode="stateHidden">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
